### PR TITLE
Add Super.so detection

### PR DESCRIPTION
**Pull request by ysm0622**
website:
- https://super.so

examples: https://super.so/showcase

- https://learn.rainbow.me/
- https://nathanclark.net/
- https://docs.basedash.com/
- https://bensomething.com/
- https://sofahq.super.site/
- https://docs.spline.design/

### CODE REVIEW

File: src/technologies/s.json [hunk 1]
```diff
@@ -4555,6 +4555,21 @@
     "scriptSrc": "plugins/super-socializer/.+?ver=([\\d.]+)\\;version:\\1",
     "website": "https://super-socializer-wordpress.heateor.com"
   },
+  "Super.so": {
+    "cats": [
+      51
+    ],
+    "description": "Build websites with Notion.",
+    "dom": "div.super-content",
```
Comment (nurbek91) [inline src/technologies/s.json:4563]: https://www.europafm.com/ 
One question on the pattern. Is this site built with this technology.
If not, then you can probably leave it, or use other options like link and img[srcset]

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!!!
Minor changes are required to bring your request to the ideal.

File: src/technologies/s.json [hunk 2]
```diff
@@ -4555,6 +4555,21 @@
     "scriptSrc": "plugins/super-socializer/.+?ver=([\\d.]+)\\;version:\\1",
     "website": "https://super-socializer-wordpress.heateor.com"
   },
+  "Super.so": {
```
[the remark this answers is not in the document]
Comment (ysm0622) [reply src/technologies/s.json]: [664c56e](https://github.com/AliasIO/wappalyzer/pull/5713/commits/664c56ed76a6e0fd220d7440773725cf789d7373)

File: src/technologies/s.json [hunk 3]
```diff
@@ -4555,6 +4555,21 @@
...
+    "cats": [
+      51
+    ],
+    "description": "Build websites with Notion.",
```
[the remark this answers is not in the document]
Comment (ysm0622) [reply src/technologies/s.json]: [664c56e](https://github.com/AliasIO/wappalyzer/pull/5713/commits/664c56ed76a6e0fd220d7440773725cf789d7373)

File: src/technologies/s.json [hunk 4]
```diff
@@ -4555,6 +4555,21 @@
...
+    "dom": "div.super-content",
+    "icon": "super.so.png",
```
[the remark this answers is not in the document]
Comment (ysm0622) [reply src/technologies/s.json]: [664c56e](https://github.com/AliasIO/wappalyzer/pull/5713/commits/664c56ed76a6e0fd220d7440773725cf789d7373)

File: src/technologies/s.json [hunk 5]
```diff
@@ -4555,6 +4555,21 @@
...
+    "implies": "Next.js",
+    "oss": false,
+    "pricing": [
+      "freemium"
```
[the remark this answers is not in the document]
Comment (ysm0622) [reply src/technologies/s.json]: [664c56e](https://github.com/AliasIO/wappalyzer/pull/5713/commits/664c56ed76a6e0fd220d7440773725cf789d7373)

File: src/technologies/s.json (hunk 1 above)
Comment (ysm0622) [reply src/technologies/s.json]: [664c56e](https://github.com/AliasIO/wappalyzer/pull/5713/commits/664c56ed76a6e0fd220d7440773725cf789d7373)

I'm not sure for this. but is it enough? 🤔

```json
    "dom": "link[href*='https://super-static-assets.s3.amazonaws.com']",
```

Comment (ysm0622) [reply src/technologies/s.json]: @nurbek91

Comment (nurbek91) [reply src/technologies/s.json]: "dom": "link[href*='super-static-assets\\.'],   img[srcset*='super-static-assets\\.']",
Hey @ysm0622 , something like this, couldnt test , currently i am not at the pc

Comment (ysm0622) [reply src/technologies/s.json]: @nurbek91 [0ad296a](https://github.com/AliasIO/wappalyzer/pull/5713/commits/0ad296a37e1e54d9ef6bed3374092d7877f8b148)

**Review by nurbek91 — APPROVED** [review]
Hey, thanks!!!

**Comment by ysm0622** [comment]
@nurbek91 Thank you for your kind review. 🙏🏻

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**